### DISCUSSION TITLE
Disabling Arsenal Complete

### DISCUSSION THREAD

**Discussion by Lasse1124** [Vdauphin/HeartsAndMinds#1654] [Question and answer]
Is there a possibility to the disable the Arsenal complete?

**Comment by Vdauphin** [comment]
Hello,

No, I would comment the line: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/int/add_actions.sqf#L223-L235

Cheers
